src/jobflow_remote/cli/types.py

```diff
@@ -337,6 +337,31 @@
     ),
 ]
 
+foreground_index_opt = Annotated[
```

Comment:
See my comment in the job controller. Do we need this option ? Maybe we can always have this in the foreground. What do you think ?

Reply:
Resolving based on your reply in the job controller.